all.json

```diff
@@ -7911,7 +7910,7 @@
 		"polkadots-drop.com",
 		"polkadots-event.com",
 		"polkadots.js.org",
-		"polkadots.live",
```

Comment:
Was this one meant to be removed? There is nothing here at this point in time.

Reply:
No only this one: polkadot-ui.authright-test.newtouch.com, I wouldn't know about the other ones

Reply:
Yes, indeed, the polkadot-ui.authright-test.newtouch.com one needs removal (as done above), just asking about polkadots.live here specifically which has also been removed here.

Reply:
I thought I only deleted the polkadot-ui one. Also seems that way in the commit too me. I didn't mean to delete any other URL

Reply:
The other one was removed/changed, see 

![image](https://user-images.githubusercontent.com/1424473/190111803-ef75c0b1-9bbe-439e-bd3c-19c4698915e2.png)

... hence my question :)